markdown/ch4.markdown

```diff
@@ -1139,27 +1126,6 @@ An individual field is accessed using dictionary syntax:
 azimuths = scan['position']['az']  # Get all azimuth measurements
 ```
 
-To construct an array such as the above from scratch, one would first
```

Comment:
I can revert, but imho it's not a very compelling example because it's out of nowhere, without using real data, and without fitting in the chapter. When I was reading through the chapter, my reaction was, "um, that was random...?" For most of the book, we don't say "if you ever wanted to do this random thing for whatever reason", we say "so now we need to do this in order to achieve such and such scientific goal". So I'm in favour of putting this e.g. in notes/scraps.txt until we can actually integrate it properly. Thoughts?